pkg/monitoring/mapping.go

```diff
@@ -97,8 +97,6 @@ func MakeConfigSetMapping(
 				"billing": map[string]interface{}{
 					"observation_payment": int64(state.Config.Billing.ObservationPayment),
 				},
-				"validator":          state.Config.Validator[:],
-				"flagging_threshold": int64(state.Config.FlaggingThreshold),
```

Comment:
@topliceanu The validator argument was removed, but flagging_threshold was moved over to the transmissions account. I'm also moving over decimals + description in a follow-up, is that ok or should changes be made so that monitoring also reads from the feed account?

Reply:
I don't know what the feed account is? Is it the same as the transmissions account? 
Yeah, it looks like all of that might need to be update after your changes are ins